scripts/modules/MsIndustryLinks/azureDeploymentPackage/azureblobstorage/connection.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
Is there a template parameters file we could provide?

Reply:
What is the reason?

Reply:
I did just realise that when you script the creation of the assets, that it wont be required.